src/main/java/nl/tudelft/scrumbledore/userinterface/GameDisplay.java

```diff
@@ -343,10 +336,8 @@ private static void prepareRenderer() {
   private static void renderStatic() {
     staticContext.clearRect(0, 0, Constants.GUIX, Constants.GUIY);
 
-    for (Platform current : currentGame.getCurrentLevel().getPlatforms()) {
-      staticContext.drawImage(new Image(sprites.get("wall-1").getPath()),
-          current.getPosition().getX(), current.getPosition().getY());
-    }
+    ArrayList<LevelElement> staticElements = currentGame.getCurrentLevel().getStaticElements();
+    renderLevelElements(staticElements, staticContext);
```

Comment:
Could also be constructed as
renderLevelElements(currentGame.getCurrentLevel().getStaticElements(), staticContext);
<review